pyproject.toml

```
@@ -48,13 +48,15 @@ sevenn = { version = "0.9.3", optional = true }
 torchdata = {version = "0.7.1", optional = true} # Pin due to dgl issue
 torch_geometric = { version = "^2.5.3", optional = true }
 ruff = "^0.5.7"
+nequip = {version = "^0.6.1", optional = true }
```

```suggestion
nequip = {version = "0.6.1", optional = true }
```
Probably best to pin like the other MLIPs to avoid API/dependency breaking changes?